server/graphql/v2/object/Host.js

```diff
@@ -727,6 +728,28 @@ export const Host = new GraphQLObjectType({
         description: 'Returns whether the host is trusted or not',
         resolve: account => get(account, 'data.isTrustedHost', false),
       },
+      hasDisputedOrders: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description: 'Returns whether the host has any disputed orders',
+        async resolve(collective) {
+          const [results] = await sequelize.query(
+            `
+            SELECT o.id FROM "Orders" o
+            JOIN "Transactions" t ON t."OrderId" = o.id
+            WHERE t."HostCollectiveId" = :hostCollectiveId AND o.status = 'DISPUTED'
+            GROUP BY o.id;
+            `,
+            {
+              type: QueryTypes.SELECT,
+              replacements: {
+                hostCollectiveId: collective.id,
+              },
+            },
+          );
+
+          return !isEmpty(results);
```

Comment:
If you're checking for emptiness, I suggest using a count query.

Reply:
You can also limit the return to 1 record.

`... LIMIT 1`

Reply:
Changed to `LIMIT 1`, good call!